### PR TITLE
relocate coverage

### PR DESCRIPTION
**Pull request by brandonc**
## Description

I'd really like to see API coverage on the README and keep it top-of-mind when receiving updates to go-tfe.

I also looked over the documented API and added some additional items:
- unchecked coverage for private registry providers et al, and run tasks integration
- checked coverage for comments

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ If you find an issue with this package, please create an issue in GitHub. If you
 
 ## Adding new functionality or fixing relevant bugs
 
-If you are adding a new endpoint, make sure to update COVERAGE.md where we keep a list of the TFC APIs that this SDK supports.
+If you are adding a new endpoint, make sure to update the coverage list in README.md where we keep a list of the TFC APIs that this SDK supports.
```
Comment (annawinkler) [inline docs/CONTRIBUTING.md:7]: Could we add a link to the coverage section in the readme (and perhaps have an anchor in the readme)?

File: README.md [hunk 2]
```diff
@@ -60,7 +60,68 @@ For complete usage of the API client, see the [full package docs](https://pkg.go
 
 ## API Coverage
 
-This API client covers most of the existing Terraform Cloud API calls and is updated regularly to add new or missing endpoints. For a complete list of what is supported/unsupported, please [refer to this page](docs/COVERAGE.md).
+This API client covers most of the existing Terraform Cloud API calls and is updated regularly to add new or missing endpoints.
```
Comment (annawinkler) [inline README.md:63]: 💖 I like this idea!

**Review by annawinkler — COMMENTED** [review]
🥳 I like this idea because initially I didn't see the file `COVERAGE.md`. I have a few suggestions for things to include in this PR.

**Review by annawinkler — DISMISSED** [review]
💖

File: README.md [hunk 3]
```diff
@@ -60,7 +60,68 @@ For complete usage of the API client, see the [full package docs](https://pkg.go
...
+- [x] State Versions
+- [x] State Version Outputs
+- [ ] Subscriptions
+- [x] Team Access
+- [x] Team Membership
+- [x] Team Tokens
+- [x] Teams
+- [x] User Tokens
+- [x] Users
+- [x] Variables
```
Comment (sebasslash) [inline README.md:108]: Add `- [x] Variable Sets` as well.

**Review by sebasslash — DISMISSED** [review]
Minor thing to add, master jedi ⬇️

**Review by sebasslash — APPROVED** [review]
👍

**Pull request merged**